packages/den/image/PinholeCameraModel.ts

```diff
@@ -131,50 +133,105 @@ export class PinholeCameraModel {
   }
 
   /**
-   * Projects a 2D image pixel to a point on a plane in 3D world coordinates a
-   * unit distance along the Z axis. This is equivalent to `projectPixelTo3dRay`
-   * before normalizing.
+   * Undoes camera distortion to map a given pixel coordinate from normalized raw image coordinates to normalized undistorted coordinates.
    *
-   * @param out - The output vector to receive the 3D point.
-   * @param pixel - The 2D image pixel coordinate.
-   * @returns `true` if the projection was successful, or `false` if the camera
-   *   projection matrix `P` is not set.
+   * @param out - The output rectified 2D normalized coordinate.
```

Comment:
I think the old comment is more clear to me. I like the `to receive` word choice.
```suggestion
   * @param out - The output vector to receive the rectified 2D normalized coordinate.
```